day2/q1.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"fmt"
+	"sync"
+)
+
+func count(input []string) []int {
+	// freq := make(map[string]int) hash map
+	freq := make([]int, 26)
+	var wg sync.WaitGroup
+	var mu sync.Mutex
```

Comment:
expectation was also to use a channel, don't see the usage of channel in this solution